### PR TITLE
Add OpenShift resources for 5.9.0

### PR DESCRIPTION
**Pull request by Chathuru**
Add OpenShift resources for 5.9.0

### CODE REVIEW

File: Template.yaml [hunk 1]
```diff
@@ -0,0 +1,731 @@
...
+
+- name: RESOURCES_REQUEST_MEMORY
+  displayName: Set a minimum memory resource limits
+  required: true
+  value: '2Gi'
+
+- name: WSO2_IS_SERVICE_ACCOUNT_NAME
+  displayName:
+  required: true
+  value: wso2svc-account
```
Comment (ThilinaManamgoda) [inline Template.yaml:731]: Please add a new line.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,27 @@
-# openshift-is
```
Comment (ThilinaManamgoda) [inline README.md:1]: Why these lines are removed?

**Review by ThilinaManamgoda — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,27 @@
...
+   `oc create secret docker-registry wso2-image-pull-secret --docker-server=docker.wso2.com --docker-username=<SUBSCRIPTION_USERNAME> --docker-password=<SUBSCRIPTION_PASSWORD>`
+
+3. Deploy WSO2 Identity Server HA cluster.
+
+   `oc process -f Template.yaml -p NFS_SERVER_IP=<NFS_SERVER_IP> -p NFS_SHARE_DATABASE=<DATABASE_NFS_SHARE_PATH> -p NFS_SHARE_USERSTORE=<USERSTORE_NFS_SHARE_PATH> -p NFS_SHARE_TENANT=<TENANT_SHARE_PATH> | oc create -f -`
+
+4. Access product management consoles.
+   Obtain the (HOST/PORT) of the route resource.
+   
+   `oc get route`
```
[the remark this answers is not in the document]
Comment (Chathuru) [reply README.md]: Fix in https://github.com/wso2/openshift-is/pull/1/commits/86e40fc1e706c2890aacee75f5fec80a41ac98a1 commit

File: Template.yaml (hunk 1 above)
Comment (Chathuru) [reply Template.yaml]: Fix in https://github.com/wso2/openshift-is/pull/1/commits/86e40fc1e706c2890aacee75f5fec80a41ac98a1 commit

File: README.md (hunk 2 above)
Comment (Chathuru) [reply README.md:1]: Fix in https://github.com/wso2/openshift-is/pull/1/commits/86e40fc1e706c2890aacee75f5fec80a41ac98a1 commit

**Pull request merged**